### DISCUSSION TITLE
Contribution's board 🌺💓👀

### DISCUSSION THREAD

**Discussion by Lumexio** [community/community#3405] [Mobile]
It would be cool to have the contribution's board from the web page in the android app, because it is nice to have a sense of advancement on the go, right now just iphone users have this feature in the ios GitHub app.

👻🥸🌺💜💙❤️📲

**Comment by PranamBhat** [comment]
Hi @Lumexio 

Can you please elaborate this need/feedback?

**Reply by Lumexio** [reply]
well, on the github website, there is a table of contributions and this is a feature that I understand is very useful for other users who like to find out about another developer and be aware of how active they are in their projects, also, for me personally it helps me to have a graphic way of my progress and activity constantly. So in the github application this feature does not exist and it would be very useful to have it available.

![Screenshot_20210223-003207.jpg](https://user-images.githubusercontent.com/55640491/108813612-9ca76300-756e-11eb-861d-bde7d75a9497.jpg)

**Reply by PranamBhat** [reply]
Hi @Lumexio 

I got it now.

Currently we have these feature's available over **GitHub** profile:


![image](https://user-images.githubusercontent.com/32120033/108815136-f1cc8e80-75d9-11eb-9a25-4017930758d9.png)



![image](https://user-images.githubusercontent.com/32120033/108815396-6acbe600-75da-11eb-977a-474fc01b706e.png)

**Reply by Lumexio** [reply]
But you have it on the app natively ? in that case, android or ios?

**Reply by PranamBhat** [reply]
Hi @Lumexio 

Yes. Everywhere 😄 

At the **GitHub** Universe event in San Francisco, the company announced that it will release official Android and iOS apps that will enable developers to reply to questions from users, read bug reports, and assign issues to other developers from their phones.

The app first launched in beta on **iOS** in **November 2019** and on **Android** in **January 2020**.

Download here :

[Android](https://play.google.com/store/apps/details?id=com.github.android)

[iOS : iPhone and iPad](https://apps.apple.com/app/github/id1477376905?ls=1)

**Reply by Lumexio** [reply]
Right, like I'm aware of his existence, but there's no contribution's board in the native android application, and that's the feature that I'm asking for :(

**Reply by Lumexio** [reply]
Pero de todas formas, muchas gracias por tu atención 🤍

**Reply by PranamBhat** [reply]
[**Check here**](https://github.blog/changelog/2021-02-02-github-for-ios-contribution-graph-widget/)

**Comment by candyho** [comment]
Hey @Lumexio , if you are on iOS we have some great news for you! We have just shipped the Contribution Graph Widget for the GitHub iOS app so that you can see your own progress on the go, right from the home screen of your iPhone or iPad. You can find more details here - https://github.blog/changelog/2021-02-02-github-for-ios-contribution-graph-widget/

**Reply by Lumexio** [reply]
Blessed 🙏 but, i'm a third world dude, and i have an android hahaha :p

**Reply by Lumexio** [reply]
Gracias 🙏🌺
